lsql/judge/signals.py

```diff
@@ -44,3 +45,60 @@ def refresh_sub_prob_achievements(sender, **kwargs):
     """Delete and check new and old NumSolvedCollectionAchievementDefinition achievements"""
     logger.debug('Signal post_save for %s %s', str(sender), str(kwargs['instance']))
     kwargs['instance'].refresh()
+
+
+def save_hints(kwargs):
+    """Create the hints of a problem an save them"""
+    for elem in kwargs['instance'].hints_info:
```

Comment:
Más compacto:

    for num_sum, description in problem.hints_info

(suponiendo que pasas problem como parámetro)